simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/StationDataHolder.java

```diff
@@ -0,0 +1,13 @@
+package com.evbox.everon.ocpp.simulator.station;
+
+import lombok.Data;
+
+@Data
+public class StationDataHolder {
```

Comment:
Having two entities PersistenceLayer and DataHolder is confusing, I'd like to avoid it